core/nmf-composites/nanosat-mo-connector/src/main/java/esa/mo/nmf/nanosatmoconnector/NanoSatMOConnectorImpl.java

```diff
@@ -92,10 +95,12 @@
     @Override
     public void init(final MonitorAndControlNMFAdapter mcAdapter) {
         super.startTime = System.currentTimeMillis();
-        HelperMisc.loadPropertiesFile(); // Loads: provider.properties; settings.properties; transport.properties
-        ConnectionProvider.resetURILinksFile(); // Resets the providerURIs.properties file
+        System.out.println(this.generateStartBanner());
```

Comment:
Use logger instead.
`System.out.println` is antipattern and makes it impossible to control and capture in Java way. If timestamp is an issue, it can be tailored in logging.properties.